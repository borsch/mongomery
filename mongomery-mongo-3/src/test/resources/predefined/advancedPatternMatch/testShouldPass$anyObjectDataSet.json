{
  "Vocabulary": [
    {
      "complexObj": {
        "f1": "v1",
        "f2": "v2",
        "f3": "v3"
      }
    }
  ]
}