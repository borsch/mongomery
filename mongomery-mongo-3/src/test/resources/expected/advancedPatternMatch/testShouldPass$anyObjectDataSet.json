{
  "Vocabulary": [
    {
      "_id": "$anyObject()",
      "complexObj": "$anyObject(3)"
    }
  ]
}